sys-boot/syslinux/syslinux-6.04_pre1-r2.ebuild

```diff
@@ -28,6 +28,7 @@ RDEPEND="sys-fs/mtools
 DEPEND="${RDEPEND}
 	dev-lang/nasm
 	>=sys-boot/gnu-efi-3.0u
+	sys-apps/util-linux:=
```

Comment:
No need for the subslot operator (`:=`) since `util-linux` doesn't have sublots. Also, if the only dependency is due to headers, a subslot operator wouldn't be warranted.

Reply:
done

Reply:
We need libuuid too, so I've added to RDEPEND.